frontend/README.md

```diff
@@ -1,34 +1,53 @@
 # The Urlist - Frontend
+
 [![Build status](https://burkeknowswords.visualstudio.com/The%20Urlist/_apis/build/status/Frontend%20Build)](https://burkeknowswords.visualstudio.com/The%20Urlist/_build/latest?definitionId=7)
 
-The frontend for this project was build with the following libraries and frameworks:
+The frontend has the following views:
+
+- **Homepage view**: this is the first page you would see when visiting the frontend. You can start the process of creating a link-bundles from here.
+
+  ![Homepage picture](docs/Images/Homepage.png)

```

Comment:
Images aren't rendering.  Did you add them to the repo?

Reply:
Yes they are included in this PR, not sure though if Github would render them from PR view. But here is a screenshot from my local branch
![image](https://user-images.githubusercontent.com/58956796/72951947-0db19100-3d45-11ea-9733-3831a9de1c58.png)